vuepress/docs/next/tutorials/devops/plugin-configuration.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebarDepth: 2
+---
+
+# Customize Deployment with Plugin Configuration Profiles
+
+This tutorial describes three methods to utilize configuration profiles in the OperatorConfigMap to specify resource allocation. This provides a simple way to customize the deployment parameters of Entando bundle plugins for improved efficiency.
+
+The three methods to insert Configuration profiles are: 
+1. As [Inline Profile](#method-1-inline-profile),
```

Comment:
As an

Reply:
I don't think you need "As" or "As an". Just list the 3 options without commas or conjunctions.